packages/image_picker/image_picker_for_web/lib/image_picker_for_web.dart

```diff
@@ -121,6 +121,9 @@ class ImagePickerPlugin extends ImagePickerPlatform {
     List<XFile> files = await getFiles(
       accept: _kAcceptImageMimeType,
       capture: capture,
+      maxWidth: maxWidth,
+      maxHeight: maxHeight,
+      imageQuality: imageQuality,
```

Comment:
Please, do not pass the image quality parameters to the getFiles method, otherwise you have to drill those down through several layers of the plugin.